.circleci/config.yml

```diff
@@ -3,13 +3,13 @@ version: 2
 jobs:
   build:
     docker:
-      - image: hashicorp/terraform:0.9.11
+      - image: hashicorp/terraform:0.10.1
```

Comment:
This really just updates the tests to show that the module supports 0.10.x, there isn't anything in the module that requires it though.  I'd say lets hold of releasing a major release until some functionality gets added that requires `0.10.x`